spec/app/ics-028-cross-chain-validation/technical_specification.md

```diff
@@ -470,18 +456,18 @@ function CreateConsumerClient(p: CreateConsumerChainProposal) {
 - **Caller**
   - Either `CreateConsumerChainProposal` (see [CCV-PCF-CCPROP.1](#ccv-pcf-ccprop1)) or `BeginBlock()` (see [CCV-PCF-BBLOCK.1](#ccv-pcf-bblock1)).
 - **Trigger Event**
-  - A governance proposal with `CreateConsumerChainProposal` as content has passed (i.e., it got the necessary votes).
+  - A governance proposal `CreateConsumerChainProposal` `p` has passed (i.e., it got the necessary votes).
 - **Precondition** 
-  - `currentTimestamp() > spawnTime`, where `spawnTime` is contained by the `CreateConsumerChainProposal`.
+  - `currentTimestamp() > p.spawnTime`.
 - **Postcondition** 
   - `UnbondingPeriod` is retrieved from the provider Staking module.
-  - A client state is created (as defined in [ICS 7](../../core/ics-002-client-semantics)).
-  - A consensus state is created (as defined in [ICS 7](../../core/ics-002-client-semantics)).
+  - A client state is created (as defined in [ICS 7](../../client/ics-007-tendermint-client)).
+  - A consensus state is created (as defined in [ICS 7](../../client/ics-007-tendermint-client)).
```

Comment:
This is part of a larger discussion outside of the scope of this PR. But it seems to me that there's nothing about Interchain Security that requires Tendermint itself. It just requires Proof of Stake. Perhaps it will be worth generalizing the spec at some point

Reply:
I agree. We could have a spec that generalizes PoS clients, and then the Tendermint client will just be a specific PoS client.